packages/gatsby-source-contentful/src/cache-image.js

```diff
@@ -1,9 +1,10 @@
-const crypto = require(`crypto`)
 const { resolve, parse } = require(`path`)
 
 const axios = require(`axios`)
 const { pathExists, createWriteStream } = require(`fs-extra`)
 
+const createContentDigest = require(`../../gatsby/src/utils/create-content-digest`)
```

Comment:
we should pass this from caller and not import directly here

Reply:
we should only import it directly for tests

Reply:
thank you for the comment! getting from the caller makes a lot more sense! I fixed all but two of these instances in previous commits.

I'm going to work on removing the import from gatsby-source-contentful and gatsby-transformer-sqip (both use the cache-image code mentioned above)